app/webpacker/controllers/dependant_select_controller.ts

```diff
@@ -0,0 +1,30 @@
+import { Controller } from "stimulus";
```

Comment:
And here is a new brand stimulus controller. 
We must find a way to test it, don't you think @Matt-Yorkley ?

Reply:
We're adding system tests in this PR: https://github.com/openfoodfoundation/openfoodnetwork/issues/7956, which I guess we'll use for the browser integration tests for the new checkout.

I'll take a look at options for Javascript unit tests...

Reply:
I think we can leave unit tests for now though, and look at it in another PR. It seems like most people use [Jest](https://jestjs.io) for unit-testing Stimulus controllers: https://shime.sh/testing-stimulus

Reply:
Yes, I used to work with Jest a lot, and I appreciated it ;)

Reply:
Great! I think we'll go in that direction. Honestly I find the Karma/Jasmine unit tests for Angular a total pain in the ass, so it would be nice to use something else :smile:

Reply:
I agree: count on me, I'll be advocate for jest ;)